test/integration/scenarios/stress/4.register_multisignature.js

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+'use strict';
+
+var Promise = require('bluebird');
+var lisk = require('lisk-js').default;
+var accountFixtures = require('../../../fixtures/accounts');
+var constants = require('../../../../helpers/constants');
+var randomUtil = require('../../../common/utils/random');
+var waitFor = require('../../../common/utils/wait_for');
+var sendTransactionsPromise = require('../../../common/helpers/api')
+	.sendTransactionsPromise;
+var getUnsignedTransaction = require('../../utils/http').getUnsignedTransaction;
+var getTransaction = require('../../utils/http').getTransaction;
+
+module.exports = function(params) {
+	describe('stress test for type 4 transactions @slow', () => {
+		var transactions = [];
+		var accounts = [];
+		var maximum = 1000;
+
+		function confirmUnsignedTransactionsOnAllNodes() {
```

Comment:
Why we don't sign any of the multisig upgrade transactions? That way we will stress the network is not consistent with how we are doing in the rest of cases. The file's name itself points to a complete registration which is not carried out.
Right now, we only perform the first phase of a multisig upgrade account. Then we only stress the queue `unsigned` and the fillPool batcher but we don't process any transaction.

Reply:
Signing the multitransactions was not part of the stress test that we already defined for type 4 transaction in lisk-core-qa repo, that is why it isn't included here, too.

I have nothing against signing the transaction as part of the stress test, though. Just wanted to keep it consistent.